app/helpers/application_helper.rb

```diff
@@ -19,12 +19,12 @@ def render_component_version(component)
   def render_example(component, example)
     content_tag(:article, id: example) do
       if File.exist? markdown_path(component, example)
-        concat render_markdown(component, example)
+        #concat render_markdown(component, example)
```

Comment:
Since this site is public facing, I'd prefer to keep it as close to a "production ready" state as possible.  Check with @mmalaker, but I don't think he'll mind the docs being there.